src/js/events/CosmosPackagesActions.js

```diff
@@ -149,6 +156,35 @@ const CosmosPackagesActions = {
     });
   },
 
+  fetchServiceDescription(packageName) {
+    RequestUtil.json({
+      contentType: getContentType("describe", "request", "v1", "service"),
+      headers: {
+        Accept: getContentType("describe", "response", "v1", "service")
+      },
+      method: "POST",
+      url: `${Config.rootUrl}/cosmos/service/describe`,
+      data: JSON.stringify({ appId: packageName }),
```

Comment:
Let's make sure we pass in `appId` and not `packageName` 👍

Reply:
Changed to `appId` and use shorthand notation for setting the key 👍